docs/UserGuide.md

```diff
@@ -464,7 +464,7 @@ Action | Format
 **Edit** | `edit INDEX [name/NAME] [phone/PHONE_NUMBER] [email/EMAIL] [home/ADDRESS] [bday/BIRTHDAY][tag/TAG]…​` | `edit 2 name/James Lee email/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]` <br> `find NUMBER`                                                    | `find James Jake` <br> `find 8651`
 **Find Tag** | `findTag TAG [MORE_TAGS]`                                                                            | `findTag Operations Outreach`
-**Edit Loan** | `editLoan INDEX AMOUNT REASON`                                                                       | `editLoan 1 -20 Buy Logistics`
+**Edit Loan** | `editLoan INDEX amt/AMOUNT reason/REASON` <br> `editLoan NAME amt/AMOUNT reason/REASON`                | `editLoan 1 amt/-20 reason/Buy Logistics` <br> `editLoan alex amt/30 reason reason/payment`
```

Comment:
Extra "reason" in `editLoan alex amt/30 reason reason/payment`

Reply:
Fixed please check